src/wp-admin/js/customize-controls.js

```diff
@@ -8992,7 +9036,6 @@
 			api( settingId, function( setting ) {
 				$.each( o.controls, function( i, controlId ) {
 					api.control( controlId, function( control ) {
-						// eslint-disable-next-line func-style
```

Comment:
`// eslint-disable-next-line func-style`

Leave this comment line in the code to silence `Expected a function declaration` errors / warnings in ESLint.

Reply:
Thanks for the explanation! Now done.